README.md

```diff
@@ -3,11 +3,47 @@ Parchment
 
 Parchment is the Interactive Fiction player for the web. To play a story with Parchment go to <https://iplayif.com>!
 
+Building Instructions
+---------------------
+
+The upstream projects are included as git submodules. You'll have to start by initializing the submodules, like this:
+
+```
+git submodule update --init --recursive
```

Comment:
I don't think recursive is needed now

Reply:
Eh, it's harmless, and without it, `link-local-emglken` won't work quite right.